finetune/base.py

```diff
@@ -244,8 +241,24 @@ def finetune(self, Xs, Y=None, batch_size=None):
                 steps_per_epoch=steps_per_epoch,
                 early_stopping_steps=self.config.early_stopping_steps,
                 eval_frequency=early_stopping_interval,
+                cache_weights_to_file=self.config.cache_weights_to_file
             )
         )
+
+        if self.config.in_memory_finetune is not None:
```

Comment:
Need to setup some kind of factory system for things like this so that every new hook doesn't need to modify base.py